x-pack/platform/plugins/shared/inference/server/chat_complete/anonymization/regex_worker_wrapper.js

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
```

Comment:
See https://piscinajs.dev/getting-started/Basic%20Usage#using-a-worker-wrapper-file-in-a-typescript-project